README.rst

```diff
@@ -37,9 +37,17 @@ Current information can always be found at the NIPY website::
 Mailing Lists
 =============
 
-Please see the developer's list::
+For questions on how to use nipy, please see NeuroStars:
```

Comment:
Sorry, my mistake--I thought this had been agreed on. Here's not the place for such a discussion, I just wanted to update things ASAP.

@matthew-brett how about using github issues? or should everything go through the mailing list?